compiler/rustc_hir/src/hir.rs

```diff
@@ -1545,10 +1549,10 @@ pub fn is_range_literal(expr: &Expr<'_>) -> bool {
             **qpath,
             QPath::LangItem(
                 LangItem::Range
-                    | LangItem::RangeTo
-                    | LangItem::RangeFrom
-                    | LangItem::RangeFull
-                    | LangItem::RangeToInclusive,
+                | LangItem::RangeTo
```

Comment:
I'm not sure why this happens. I ran `./x.py fmt`, has that changed its functionality?